metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -331,7 +333,7 @@ parenthesized_expression:   "(" WS_INLINE? expression WS_INLINE? ")"
 
 not_implemented_expression: "..."
 
-invoke_expression:  "#"? primary_expression "(" NEWLINE? argument_list? NEWLINE? ")"
+invoke_expression:  "#"? primary_expression "(" NEWLINE? keyword? argument_list? NEWLINE? ")"
```

Comment:
On second pass, this doesn't look right - `keyword` includes a lot of other things (https://github.com/mohdsiddique/datahub/blob/master+powerbi-aws-redshift-lineage/metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule#L169), not all of which apply here

I think you should expand the `expression` or `argument_list` definition instead

Reply:
done